new-dti-website/src/app/globals.css

```diff
@@ -192,3 +192,34 @@
   color: black;
   background-color: #fefefe;
 }
+
+summary {
+  padding-top: 1rem;
+  padding-bottom: 1rem;
+  list-style: none;
+  display: flex;
+  align-items: center;
+  font-weight: bold;
+  justify-content: space-between;
+  padding-right: 1rem;
+}
+
+summary::after {
+  content: '';
+  width: 13px;
+  height: 7px;
+  background: url('/icons/dropdown.svg');
+  background-size: cover;
+}
+
+details[open] > summary::after {
+  rotate: 180deg;
+}
+
+details {
+  user-select: none;
```

Comment:
Clicking on the accordian a few times in a row leads to some of the summary text getting highlighted, which is a regression from previous behavior. So I made the details not selectable.

Reply:
I do get ur point tho andrew and it is a shame that using `<details>` leads to this regression, so if you feel like that adding `user-select: none;` is worth it in this case, I'll defer to you here.

but let's also make sure that this is an exception bc i dont think we want to make this behavior standard

Reply:
Yeah, I had a feeling it was bad practice. I reverted that part! I agree with you that rapidly clicking might be something we do when testing it, but in practice, we wouldn't do that so I wouldn't consider it to be a "primary task purpose of a control".

Reply:
ok, sounds good then!